packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -187,3 +201,16 @@ class ${class_name}:
 
         for response in self._get_stub().Measure(request):
             yield self._deserialize_response(response)
+
+    def register_pin_map(self, pin_map_path: str) -> str:
```

Comment:
`register_pin_map` method is used only to update the pin map alone. Can we proceed keeping this as method or make it as property to update the pin map?
May I know your suggestions. @dixonjoel @mshafer-NI

Reply:
Seems reasonable to use a method to me.